isisd/isis_cli.c

```diff
@@ -594,24 +594,50 @@ void cli_show_isis_overload(struct vty *vty, struct lyd_node *dnode,
 }
 
 /*
- * XPath: /frr-isisd:isis/instance/attached
+ * XPath: /frr-isisd:isis/instance/attach-send
  */
-DEFPY_YANG(set_attached_bit, set_attached_bit_cmd, "[no] set-attached-bit",
```

Comment:
What happened to set-attached-bit?  We *must* Accept this command still.  People have this in their configuration right?    What happens to their setup if you remove this command?  Is this what we want to do?

Reply:
What would you recommend that I do with the command?   Since the attached-bit feature didn't work correctly I'm not sure how much this command would have been used.   The default now is for attached-bit to be on  (how cisco and juniper work),  where as the original implementation you had to turn it on.    Do you want two ways to configure the attached bit?    Is there a way to obsolete a command?   If we keep this command would you want me to actually hook it into the new feature,  make it a second command for the sending or receiving of attached-bit?

Reply:
I really had no idea how the command worked.  Let's have this discussion in todays tech meeting.

Reply:
Add back old set-attached-bit command and provided error log indicating it is deprecated and to use the new attached-bit [send | receive] commands